wire-ios-sync-engine/Source/Data Model/Conversation+AccessMode.swift

```diff
@@ -275,8 +266,8 @@ internal struct WirelessRequestFactory {
         }
         return .init(getFromPath: "/conversations/\(identifier)/features/conversationGuestLinks", apiVersion: apiVersion.rawValue)
     }
-
-    static func createLinkRequest(for conversation: ZMConversation, apiVersion: APIVersion) -> ZMTransportRequest {
+    // Optional password in this method as a reference and then pass it all the way up to the cell
```

Comment:
I don't quite understand this comment, could you clarify it please?

Reply:
Do we still need this method if we have the action now?

Reply:
We don't I deleted it as well as their respected tests. https://github.com/wireapp/wire-ios/commit/de595db36a92c05a68dd639e47b26fb27d8e869c